packages/modules/hrms/src/components/FormComposer.js

```diff
@@ -0,0 +1,90 @@
+import React, { useMemo } from "react";
```

Comment:
Use the formcomponent in react-components lib, incase this need any enhancements use the controlled component